upstairs/src/lib.rs

```diff
@@ -2935,6 +2935,12 @@ struct Downstairs {
      */
     live_repair_aborted: ClientData<usize>,
 
+    /**
+     * Times we skipped repairing a downstairs because we are running
+     * as read_only.
+     */
+    rop_lr_skipped: ClientData<usize>,
```

Comment:
It's not required that this is a read-only parent, just that it's read-only. Maybe `ro_lr_skipped` instead?

Reply:
Yeah, that's a better name.